python/lsst/ts/standardscripts/maintel/home_both_axes.py

```diff
@@ -75,6 +75,10 @@ def get_schema(cls):
                     description: Ignore the m1m3 component?
                     type: boolean
                     default: false
```

Comment:
You might want to remove the default value here or it will always trigger the deprecation warning. Also, please add a note in the description that this is going to be deprecated.